services_app/src/androidTest/java/org/opendatakit/database/service/GroupsPermissionTest.java

```diff
@@ -858,7 +861,7 @@ public void testGroupAuthorizationAnonymousFailureLocked_ExpectSomeFailures() {
          assertEquals(1, table.getNumberOfRows());
          row = table.getRowAtIndex(0);
 
-         assertEquals(Long.valueOf(2), row.getDataByKey(COL_INTEGER_ID));
+         assertEquals(Long.valueOf(COL_INTEGER_VALUE_TWO), row.getDataByKey(COL_INTEGER_ID));
```

Comment:
Why not make the constant an Object Long?

Reply:
Hmmmm. Okay then. I used an int because we were initially populating the contetnt values with them.